src/main_thread/decrypt/content_decryptor.ts

```diff
@@ -330,6 +371,14 @@ export default class ContentDecryptor extends EventEmitter<IContentDecryptorEven
     this.trigger("stateChange", this._stateData.state);
   }
 
+  /**
+   * Returns the list of supported codecs with the keysystem configuration.
+   * @returns {ICodecSupportList}: The list of supported codecs.
```

Comment:
In JSDoc, we historically avoided to use TypeScript type. Here it could be `Object`
The point was that its JS-compiled version could also have some minor type information